tests/testsuite/check.rs

```diff
@@ -724,6 +724,10 @@ fn check_artifacts() {
 
 #[test]
 fn short_message_format() {
+    if !is_nightly() {
+        // This can be removed once 1.30 is stable (rustdoc --error-format stabilized).
```

Comment:
The message here is wrong. This relies on `rustc --error-format` which is stable as of 1.28, but we're (currently) using 1.27.2.

I propose we upgrade to that CI setup to 1.28 and remove this guard.

Reply:
Done. Can we also remove this from `doc::short_message_format`?

Reply:
That one not till 1.30.